deps/verifier/src/tdx/quote.rs

```diff
@@ -506,7 +509,60 @@ pub async fn ecdsa_quote_verification(quote: &[u8]) -> Result<()> {
         }
     }
 
-    Ok(())
+    let tcb_status_desc = SgxQlQvResultWrapper(quote_verification_result).as_str();
+
+    let advisory_ids: Vec<Value> = get_sa_list(&supp_data.sa_list).iter()
+        .map(|s| Value::String(s.to_string()))
+        .collect();
+
+    let mut claims_map = Map::new();
+
+    claims_map.insert(
+        "tcb_status".to_string(),
+        Value::String(tcb_status_desc.to_string()),
+    );
+    claims_map.insert("advisory_ids".to_string(), Value::Array(advisory_ids));
+
+    Ok(claims_map)
```

Comment:
`az-vtpm-tdx` uses this result too (in the future SGX too if we de-duplicate this function implementation). Is it OK for that verifier to change to a serde output?

Reply:
Added custom claims also to az_tdx_vtpm

Reply:
Currently az_vtpm is already extending claims using similar approach so should be ok in my opinion